### PR TITLE
Draft documentation of results

### PR DESCRIPTION
**Pull request by e-wu**

### CODE REVIEW

File: RESULTS.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+The Journey Map blocks also describe what the user should expect depending on which "mode" the application is. The two available modes are "Conversation mode ON" which indicates that the user is speaking using the microphone to navigate the application (the screen on the left represents the layout for this mode), or "Conversation Mode OFF" which uses the buttons presented on the screen to navigate the application (the screen on the right represents the layout for this mode).
+
+The final component of a Journey Map block is the "Technical Considerations" section, this outlines the technical considerations and if they were achieved or not. Like with the above screens these too have an icon depicting if the implementation of the technical components were successful or unsuccesful. 
+
+
+## Findings
+
+### WebXR
+1. WebXR is currently a Work in Progress standard, but working towards standardization. Due to this the support for WebXR in Operating Systems and mainstream browsers is limited.
```
Comment (e-wu) [inline RESULTS.md:68]: @gdifrusc 
think it should be "... are limited"?

File: RESULTS.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+3. Limited support of the features in the prototype were offerred to iOS users using the WebXR Viewer application released by Mozilla. As well as for development Mozilla released an extension that could be installed in Chrome and Firefox to simulate the WebXR environment. However both of these applications were limited in that they could not interact with the BabylonJS GUI elements.
+ 
+### BabylonJS
+1. Due to Security policies in Chrome, the AudioContext is not allowed to start until user input/interaction is done with the webpage. This meant that the prototype could not speak an initial welcome message, and was remedied by manually unlocking the AudioContext engine before the welcome message was played.
+2. GUI Text Input does not function properly on a mobile. This is a currently known issue within the BabylonJS documentation, the Text Input element cannot invoke the onscreen keyboard so as a workaround it sends an alert window to get the input. This however causes the AR world to freeze as on mobile this prompt window does not get sent to the front above the AR world.
```
Comment (e-wu) [inline RESULTS.md]: GUI Text input does not function properly on mobile.

Comment (e-wu) [reply RESULTS.md]: This however causes the AR world to freeze on a mobile device as the window prompt does not retain focus while in the AR world. (not sure if this accurate sentence)

File: RESULTS.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### Possible Improvements to Prototype
+Quality of life improvements could be made to the prototype in its current state such as performance increases on a response from the server. This would allow for the time spent processing a request to be diminished but not by much since most of the delay in introduced waiting for a response from the cloud services rather than the API and WebXR itself. 
+
+Another improvement could be the addition of a skip button, which would allow users to skip the response if they know that the response is not what they are looking for, similar to interrupting an agent in a service centre to clarify a question. This would avoid a user being stuck listening to a response that they know is irrelevant for the entirity of the text to speech response.
+
+## References
+TODO: Link the issues as references and the babylonjs documentation.
```
Comment (e-wu) [inline RESULTS.md]: If references available add, otherwise drop

**Review by e-wu — COMMENTED** [review]
added comments

File: RESULTS.md (hunk 2 above)
Comment (gdifrusc) [reply RESULTS.md]: Updated the document with the suggested edits.

File: RESULTS.md (hunk 3 above)
Comment (gdifrusc) [reply RESULTS.md]: Put in direct links to resources instead of a references section.

**Pull request merged**